api/db-cdr/prep-tables/make-prep-survey.py

```diff
@@ -0,0 +1,455 @@
+                                           min_value, max_value, item_flags, 'controlled')
+
+                    add_answers(controlled_writer, choices, question_code, question_id_controlled
+                                , None, None, item_flags, 'controlled', long_code_to_short_code, answer_suppression)
+
+                if item_flags["REGISTERED_QUESTION_SUPPRESSED"] != 1:
+                    question_id_registered = id_registered
+                    write_row(registered_writer, parent_id_registered, question_code, question_label
+                              , 'QUESTION', None, None, item_flags, 'registered', long_code_to_short_code)
+
+                    if is_numeric_answer:
+                        add_numeric_answer(registered_writer, question_id_registered,
+                                           min_value, max_value, item_flags, 'registered')
+
+                    add_answers(registered_writer, choices, question_code, question_id_registered
+                                , None, None, item_flags, 'registered', long_code_to_short_code, answer_suppression)
+
+    # copy local output files and upload it to the bucket
+    storage_client = storage.Client.from_service_account_json('../../sa-key.json')
+    bucket = storage_client.get_bucket('all-of-us-workbench-private-cloudsql')
+    files = [f for f in listdir(home_dir) if isfile(join(home_dir, f))]
+    for f in files:
+        blob = bucket.blob('cb_prep_tables/redcap/output/' + f)
+        blob.upload_from_filename(home_dir + "/" + f)
+
+    # when done remove directory and all files
+    shutil.rmtree(home_dir)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(prog='make-prep-survey',
+                                     usage='%(prog)s --project <project> '
+                                           '--dataset <dataset> '
+                                           '--date <date>',
+                                     description='Generate 3 csv files')
+    parser.add_argument('--project', type=str, help='project name',
+                        required=True)
+    parser.add_argument('--dataset', type=str, help='dataset name',
+                        required=True)
+    parser.add_argument('--date', type=str, help='date for input file',
+                        required=True)
+    arg = parser.parse_args()
+    return arg.project, arg.dataset, arg.date
+
+
+def read_csv(file):
+    blob = get_blob(file)
+    blob = blob.download_as_string()
+    blob = blob.decode('utf-8')
+    blob = StringIO(blob)  # tranform bytes to string
+    return csv.DictReader(blob)  # then use csv library to read the content
+
+
+def files_exist(date):
+    for name in surveys:
+        file = name + '_' + date + '.csv'
+        blob = get_blob(file)
+        if not blob.exists():
+            raise Exception(file +
+                            " does not exist!")
+
+
+def not_row_to_skip(row):
```

Comment:
this is a bad method name. it doesn't describe itself well. if you named it "skip_row" then if it returned true/false it makes more sense. if (skip_row)....